File: app/src/main/java/com/time/table/PagerAdapter.java
package com.time.table;
/*
 * Created by will.spitzer on 01/06/2018.
 */

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;

public class PagerAdapter extends FragmentStatePagerAdapter {

    private int mNumOfTabs;

    public PagerAdapter(FragmentManager fm, int NumOfTabs) {
        super(fm);
        this.mNumOfTabs = NumOfTabs;
    }

    @Override
    public Fragment getItem(int position) {
        switch (position) {
            case 0:
                return new TabFragment1();
            case 1:
                return new TabFragment2();
//            case 2:
//                return new TabFragment3();
//            case 3:
//                return new TabFragment4();
//            case 4:
//                return new TabFragment5();
            default:
                return new TabFragment1();
        }

    }

    @Override
    public int getCount() {
        return mNumOfTabs;
    }
}

File: app/src/main/java/com/time/table/PagerAdapter.kt
package com.time.table

/*
 * Created by will.spitzer on 01/06/2018.
 */

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter

class PagerAdapter(fm: FragmentManager, private val mNumOfTabs: Int) : FragmentStatePagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        return when (position) {
            0 -> TabFragment1()
            1 -> TabFragment2()
            2 -> TabFragment3()
            3 -> TabFragment4()
            4 -> TabFragment5()
            else -> TabFragment1()
        }

    }

    override fun getCount(): Int {
        return mNumOfTabs
    }
}
